### PR TITLE
Update to exception list documentation in Polaris

### PR DESCRIPTION
**Pull request by pam-keesey**
Updated the Polaris exception list documentation per this conversation in Slack: https://shopify.slack.com/archives/C4Y8N30KD/p1660789923656029

### CODE REVIEW

**Comment by pam-keesey** [comment]
Hoping we can resolve any issues and merge this change. Adding Selene as a reviewer in case I've missed anything.

File: polaris.shopify.com/content/components/exception-list/index.md [hunk 1]
```diff
@@ -34,6 +34,21 @@ Exception lists should:
 - Have a description (a title is optional)
 - Be concise
 
+Punctuation for exception lists should follow [list item guidelines](https://polaris.shopify.com/components/list). Standalone exceptions (a single sentence or sentence fragment) do not need end punctuation. Titles are optional and can be used as a header or as an inline label introducing the description.
```
Comment (selenehinkley) [inline polaris.shopify.com/content/components/exception-list/index.md:37]:
```suggestion
Punctuation for exception lists should follow [list item guidelines](https://polaris.shopify.com/components/list). Standalone exceptions (a single sentence or sentence fragment) don’t need end punctuation. Titles are optional and can be used as a header or as an inline label introducing the description.
```

**Review by selenehinkley — APPROVED** [review]
Just a small edit to add a contraction, otherwise LGTM! Thanks for making another contribution, @pam-keesey!

**Comment by pam-keesey** [comment]
Hi @gwyneplaine! Hoping we can resolving any remaining issues and push this live. Thanks!

**Pull request closed without merging**